narayana-release-process.sh

```diff
@@ -13,20 +13,18 @@
 # 3 arguments: `./narayana-release-process.sh CURRENT NEXT WFLYISSUE`
 
 command -v ant >/dev/null 2>&1 || { echo >&2 "I require ant but it's not installed.  Aborting."; exit 1; }
-command -v awestruct >/dev/null 2>&1 || { echo >&2 "I require awestruct (http://awestruct.org/getting_started) but it's not installed.  Aborting."; exit 1; }
 read -p "Have you created a WFLY issue at https://issues.jboss.org/secure/CreateIssue.jspa and have the number?" WISSUEOK
 if [[ $WISSUEOK == n* ]]
 then
   exit
 fi
 if [ $# -ne 4 ]; then
-  echo 1>&2 "$0: usage: CURRENT NEXT WFLYISSUE RSYNC_HOST"
+  echo 1>&2 "$0: usage: CURRENT NEXT WFLYISSUE"
   exit 2
 else
   CURRENT=$1
   NEXT=$2
   WFLYISSUE=$3
-  RSYNC_HOST=$4
 fi
 
 if [[ $(uname) == CYGWIN* ]]
```

Comment:
Is docker still part of this process? Maybe it can be removed?

Reply:
Sorry, the docker aspect is in the information that appears collapsed in the view (as in you haven't changed it yet)

Reply:
So this bit : https://github.com/marcosgopen/narayana/blob/JBTM-3888/narayana-release-process.sh#L28-L40

Reply:
Oh, I see. I think that is for windows users, Are you sure we can delete that section?

Reply:
That section is to do with docker, which IIUC we are no longer using at release time? So I would say yes, but please check my reasoning.